spec/lita/handlers/deliveries_notification_handler_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+require 'spec_helper'
+
+describe Lita::Handlers::DeliveriesNotificationHandler, lita_handler: true do
+
+  before { allow(HTTParty).to receive(:post) }
+
+  it { is_expected.not_to route('How are you doing') }
+  it { is_expected.not_to route('To code or not to code, that is a question') }
+  it { is_expected.not_to route('How do you turn this one?') }
```

Comment:
Seems a bit excessive to have so many of these... ?

Reply:
hahaha its true, I just copy and paste from the other, but I will remove some 👍